crates/polars-plan/src/logical_plan/optimizer/simplify_expr.rs

```diff
@@ -125,7 +125,8 @@ impl OptimizationRule for SimplifyBooleanRule {
                 AExpr::Literal(LiteralValue::Boolean(true))
             ) =>
             {
-                Some(expr_arena.get(*right).clone())
+                // We alias because of the left-hand naming rule.
+                Some(AExpr::Alias(*right, "literal".into()))
```

Comment:
Maybe we should set a `static LITERAL_NAME: str` somewhere.

Reply:
Yeah this is not ideal, but when we do `rg -i literal` should pick up all cases we have hardcoded literal now.